src/components/MainForm/Mainform.tsx

```diff
@@ -71,48 +78,58 @@ export default function Mainform() {
   };
   const [Fname, setFName] = useState("");
   const [Lname, setLName] = useState("");
-  
+
   return (
     <>
       <div className="Main-div" style={{}}>
         <div className="FormDiv">
           <Header />
           <form onSubmit={handleSubmit(FormSubmitHandler)}>
-            <TextField
-              fullWidth
-              id="fullWidth"
-              label="First Name"
-              variant="outlined"
-              value={Fname}
-              {...register("firstName", { required: true })}
-            
-              onInput={(evt: any) => {
-                const caps =
-                  evt.target.value.charAt(0).toUpperCase() +
-                  evt.target.value.slice(1);
-                setFName(caps);
-              }}
-              // ref={register}
-            />
-            <TextField
-              fullWidth
-              id="outlined-basic"
-              label="Last Name"
-              variant="outlined"
-              value={Lname}
-              {...register("lastName", { required: true })}
-              style={{ marginBottom: "20px" }}
-              onInput={(evt: any) => {
-                const caps =
-                  evt.target.value.charAt(0).toUpperCase() +
-                  evt.target.value.slice(1);
-                setLName(caps);
+            <Box
+              component="form"
+              sx={{
+                "& .MuiTextField-root": { m: 0.8, width: "17ch" },
               }}
-            />
-            <br />
-
-            <Button onClick={handleClickOpen}>
-              <span className="iconify iconify-main" data-icon="cif:us"></span>
+              noValidate
+              autoComplete="off"
+            >
+              <TextField
+                id="outlined-size-normal"
+                label="First Name"
+                value={Fname}
+                style={{ marginRight: "5.75%" }}
+                {...register("firstName", { required: true })}
+                onInput={(evt: any) => {
+                  const caps =
+                    evt.target.value.charAt(0).toUpperCase() +
+                    evt.target.value.slice(1);
+                  setFName(caps);
+                }}
+              />
+              <TextField
+                id="outlined-size-normal"
+                label="Last Name"
+                value={Lname}
+                {...register("lastName", { required: true })}
+                style={{ marginBottom: "20px", marginLeft: "6.5px" }}
+                onInput={(evt: any) => {
+                  const caps =
```

Comment:
why do you need to use this props?
if you register it then why put values in component states?